server.go

```diff
@@ -931,6 +931,97 @@ func (sp *serverPeer) OnGetCFilter(p *peer.Peer, msg *wire.MsgGetCFilter) {
 	sp.QueueMessage(filterMsg, nil)
```

Comment:
I can't comment at the exact line so I leave it in here:

At line 64 `maxProtocolVersion` needs to be changed to the latest wire protocol version.